src/pretix/base/forms/questions.py

```diff
@@ -1250,6 +1258,51 @@ def __init__(self, *args, **kwargs):
                 else:
                     v.widget.attrs['autocomplete'] = 'section-invoice billing ' + autocomplete
 
+        self.fields['is_business'].widget.attrs['data-trigger-address-info'] = 'on'
+        self.fields['transmission_type'] = forms.ChoiceField(
+            label=_('Invoice transmission method'),
+            choices=[
+                (t.identifier, t.public_name) for t in get_transmission_types()
+            ]
+        )
+        self.fields['transmission_type'].widget.attrs['data-trigger-address-info'] = 'on'
```

Comment:
as this field is implicitly marked as required, invoice address fields will be switched to required by addressform.js (isAnyRequired will be true).

easiest fix is to move this block before line 1215 `if not self.address_required or self.all_optional:` so it gets affected by the `required = False` magic.